x-pack/plugin/esql/src/test/java/org/elasticsearch/xpack/esql/expression/function/AbstractFunctionTestCase.java

```diff
@@ -191,7 +191,6 @@ protected static List<TestCaseSupplier> anyNullIsNull(
         ExpectedType expectedType,
         ExpectedEvaluatorToString evaluatorToString
     ) {
-        typesRequired(testCaseSuppliers);
```

Comment:
It's now impossible to build test cases without types so this test doesn't do anything.